dataplane/dplanerc/routes.go

```diff
@@ -229,6 +230,67 @@ func (ni *Reconciler) createNextHop(ctx context.Context, hop *dpb.NextHop) (uint
 		}
 		log.Infof("created gue actions: %v", actReq)
 	}
+	if len(hop.GetHeaders().GetHeaders()) > 0 {
```

Comment:
This function is becoming lengthy.. should we move the new code in a function?

Reply:
added todo